sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/namedExpressions.scala

```diff
@@ -85,6 +85,7 @@ trait NamedExpression extends Expression {
    *    e.g. top level attributes aliased in the SELECT clause, or column from a LocalRelation.
    * 2. Seq with a Single element: either the table name or the alias name of the table.
    * 3. Seq with 2 elements: database name and table name
+   * 4. Seq with 3 elements: catalog name, database name and table name
```

Comment:
namespace instead?

Reply:
> namespace instead?

 Sorry for the late response. You mean `catalog namespace`?

Reply:
I think he meant to document the namespace support in catalogV2, which is related but I think can be done separately.

Reply:
Yea, it looks fine as it is.